checker_client.py

```diff
@@ -30,33 +41,54 @@ def connection() -> str:
             roof = "S"
             curtain_west = "S"
             curtain_east = "S"
-
+            panel = "S"
+            power = "S"
+            light = "S"
+            aux = "S"
+            
             for k, value in values.items():
                 if value:
                     if k == "RO":
                         roof = "O"
                     elif k == "RC":
                         roof = "C"
-                    elif k == "RS":
-                        roof = "S"
+                        
+                    elif k == GuiKey.PANEL_ON:
+                        panel = "A"
+                    elif k == GuiKey.PANEL_OFF:
+                        panel = "S"
+
+                    elif k is GuiKey.POWER_ON:
+                        power = "A"
+                    elif k is GuiKey.POWER_OFF:
+                        power = "S"
+
+                    elif k is GuiKey.LIGHT_ON:
+                        light = "A"        
+                    elif k is GuiKey.LIGHT_OFF:
+                        light = "S"
+
+                    elif k is GuiKey.AUX_ON:
+                        aux = "A"
+                    elif k is GuiKey.AUX_OFF:
+                        aux = "S"
 
                     elif k == "WO":
                         curtain_west = "O"
                     elif k == "WC":
                         curtain_west = "C"
                     elif k == "WS":
                         curtain_west = "S"
-
                     elif k == "EO":
                         curtain_east = "O"
                     elif k == "EC":
                         curtain_east = "C"
                     elif k == "ES":
                         curtain_east = "S"
 
-            code = roof + curtain_west + curtain_east
+                    code = roof + panel + power + light + aux + curtain_west + curtain_east
```

Comment:
check if is better to keep it outside the for loop to have continuously update from the gpio even without touching the buttons